activesupport/lib/active_support/digest.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+module ActiveSupport
+  class Digest
```

Comment:
Should this class be marked as private with `# :nodoc:`? End users won't need to call it directly.